scene/2d/animated_sprite_2d.cpp

```diff
@@ -72,6 +74,18 @@ bool AnimatedSprite2D::_edit_use_rect() const {
 	}
 	return t.is_valid();
 }
+
+void AnimatedSprite2D::get_argument_options(const StringName &p_function, int p_idx, List<String> *r_options) const {
+	if (p_idx == 0 && p_function == "play" && frames.is_valid()) {
+		const String quote_style = EDITOR_GET("text_editor/completion/use_single_quotes") ? "'" : "\"";
```

Comment:
In `master`, I'm pretty sure that I refactored this code so that the quoting style preference is handled in the script editor code itself. So you shouldn't need this, and thus not need to add an editor include in `scene/` which breaks encapsulation #53295.